agent/engine/testdata/test_tasks/sleep5TaskCgroup.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
If the `CPULimit` and `Memory` are the only missing fields in the `sleep5.json`, can we reuse the `sleep5.json` and modify the field after loading the task. We don't want every kind of task to have a json file in this directory for a unit test.

Reply:
I would prefer to keep this `json` as it makes testing cleaner. 
It's also going to be used by integ tests